src/fiatconnect/TransferStatusScreen.tsx

```diff
@@ -133,12 +133,31 @@ function SuccessOrProcessingSection({
     | FiatExchangeEvents.cico_fc_transfer_success_view_tx
     | FiatExchangeEvents.cico_fc_transfer_processing_view_tx
 
+  const getSettlementTimeString = (settlementEstimation: SettlementEstimation) => {
```

Comment:
The semantics of this switch and the switch in `PaymentMethodSection.tsx` look identical to me, save for the actual translation strings that are mapped to by the `DESCRIPTION_STRINGS` and `SETTLEMENT_TIME_STRINGS` objects respectively. Could these be combined and parameterized by the translation string mapping?